python/cugraph/cugraph/tests/test_property_graph.py

```diff
@@ -338,7 +352,13 @@ def test_num_vertices(df_type):
                        property_columns=None)
 
     assert pG.num_vertices == 9
+    assert pG.get_num_vertices() == 9
+    assert pG.get_num_vertices('merchants') == 5
+    assert pG.get_num_vertices('users') == 4
+    assert pG.get_num_vertices('taxpayers') == 7  # These nodes have two types
```

Comment:
TL;DR - [this PR](https://github.com/rapidsai/cugraph/pull/2419) will remove "taxpayers", so it might be easier to just remove these assertions here too.

There was an offline discussion (sorry for not including the outcome on GH) where it was decided that the "taxpayer" input was considered illegal input, since it results in the same vertex showing up as two different types.  I think we decided that users should format the input to have a 1:1 for types and vertex IDs, and entities that show up with multiple properties (eg. "merchants" and "taxpayers") should format their input to have bool columns such as "is_merchant" and "is_taxpayer" instead, for example.

Reply:
PG should eventually enforce vertex id uniqueness, but that should not hold up this PR.